sql: change column in tenant_usage table

RaduBerinde opened the pull request:
The tenant usage table currently has the "last update" timestamp as an
instance-only field. This field is necessary for the per-tenant state
as well (it corresponds to the current bucket level).

This change renames the `instance_last_update` column to `last_update`
and moves it up to the group of columns that apply to all rows.

Since this table was introduced only on `master`, we don't add a new
migration.

Release justification: category 2, code only used for Serverless.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/69253)
<!-- Reviewable:end -->

ajwerner reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/69253#-:-Mhoh5k30DkFpyOpExnx:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @andy-kimball)



<!-- Sent from Reviewable.io -->

RaduBerinde committed:
sql: change column in tenant_usage table

The tenant usage table currently has the "last update" timestamp as an
instance-only field. This field is necessary for the per-tenant state
as well (it corresponds to the current bucket level).

This change renames the `instance_last_update` column to `last_update`
and moves it up to the group of columns that apply to all rows.

Since this table was introduced only on `master`, we don't add a new
migration.

Release justification: category 2, code only used for Serverless.

Release note: None

RaduBerinde commented:
TFTR!

bors r+
